src/app/Dashboard/AddCard.tsx

```diff
@@ -0,0 +1,139 @@
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+import * as React from 'react';
+import {
+  Bullseye,
+  Button,
+  Card,
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateIcon,
+  EmptyStateVariant,
+  Select,
+  SelectOption,
+  Title,
+} from '@patternfly/react-core';
+import { Wizard, WizardStep } from '@patternfly/react-core/dist/js/next';
+import { PlusCircleIcon } from '@patternfly/react-icons';
+import { useDispatch } from 'react-redux';
+import { StateDispatch } from '@app/Shared/Redux/ReduxStore';
+import { addCardIntent } from '@app/Shared/Redux/DashboardConfigActions';
+import { DashboardCards, getConfigByTitle } from './Dashboard';
+
+interface AddCardProps {}
+
+export const AddCard: React.FunctionComponent<AddCardProps> = (props: AddCardProps) => {
+  const [showWizard, setShowWizard] = React.useState(false);
+  const [selection, setSelection] = React.useState(DashboardCards[0].title);
+  const [selectOpen, setSelectOpen] = React.useState(false);
+  const dispatch = useDispatch<StateDispatch>();
+
+  const options = React.useMemo(() => {
+    return [
+      ...DashboardCards.map((choice, idx) => (
+        <SelectOption key={idx} value={choice.component.name}>
+          {choice.title}
+        </SelectOption>
+      )),
+    ];
+  }, [DashboardCards]);
+
+  const handleToggle = React.useCallback(
+    (isOpen) => {
+      setSelectOpen(isOpen);
+    },
+    [setSelectOpen]
+  );
+
+  const handleSelect = React.useCallback(
+    (_, selection) => {
+      setSelection(selection);
+      setSelectOpen(false);
+    },
+    [setSelection, setSelectOpen]
+  );
+
+  const handleAdd = React.useCallback(() => {
+    setShowWizard(false);
+    dispatch(addCardIntent(getConfigByTitle(selection).component.name));
+  }, [dispatch, selection]);
+
+  const handleStart = React.useCallback(() => {
+    setShowWizard(true);
+  }, [setShowWizard]);
+
+  const handleStop = React.useCallback(() => {
+    setShowWizard(false);
+  }, [setShowWizard]);
+
+  if (!showWizard) {
+    return (
+      <>
+        <Card>
+          <Bullseye>
+            <EmptyState variant={EmptyStateVariant.large}>
+              <EmptyStateIcon icon={PlusCircleIcon} />
+              <Title headingLevel="h2" size="md">
+                Add a new card
+              </Title>
+              <EmptyStateBody>
+                Cards added to this Dashboard layout present information at a glance about the selected target. The
+                layout is preserved for all targets viewed on this client.
+              </EmptyStateBody>
+              <Button variant="primary" onClick={handleStart}>
+                Add
+              </Button>
+            </EmptyState>
+          </Bullseye>
+        </Card>
+      </>
+    );
+  } else {
+    return (
+      <>
+        <Wizard onClose={handleStop} onSave={handleAdd}>
+          <WizardStep id="card-type-select" name="Card Type">
```

Comment:
Looks very good with the wizard :D 

Just noticed that our current patternfly core does not fully support composition pattern yet

> If you seek a wizard solution that allows for more composition, see the [React next](https://www.patternfly.org/v4/components/wizard/react-next) tab.

Seems like the last step should have `Finish` as the accept but now its showing Next. Do you think we should still keep this or refactor to use `steps` props on `Wizard`?